code/modules/customitems/item_defines.dm

```diff
@@ -830,3 +830,31 @@
 	new /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass(src)
 	new /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass(src)
 	new /obj/item/weapon/reagent_containers/food/drinks/bottle/fluff/moonshine(src)
+
+/obj/item/clothing/suit/toggle/lasshy_wings //Ssscratches: Lasshy
+	name = "fairy wings"
+	desc = "Fake but Real(TM) Fairy wings! They even flap a bit."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "lasshy_wings"
+	item_state = "lasshy_wings"
+	actions_types = list(/datum/action/item_action/toggle_wings)
+	var/icon/mob
+
+/obj/item/clothing/suit/toggle/lasshy_wings/update_icon(var/mob/living/carbon/human/user) //holy pickles, copied over from /obj/item/clothing/head/kitty
+	if(!istype(user))
+		return
+	var/obj/item/organ/external/head/head_organ = user.get_organ("head")
+	mob = icon("icon" = 'icons/mob/suit.dmi', "icon_state" = icon_state)
```

Comment:
This variable isn't really accurately named, and it's a predefined path, so the syntax highlighting is fucked up. I would recommend changing it to `blended_icon`. It also doesn't really need to be defined as a static part of the path, why not just declare and initialize a new icon in this proc? It's not being used anywhere in the future.